src/index.html

```diff
@@ -1,17 +1,176 @@
+            id="age"
+            type="number"
+            name="age"
+            value="12"
+          >
+        </label>
+        <label class="label" for="birthdate"
+          >Full date of birth:
+          <input
+            required
+            id="birthdate"
+            type="date"
+            name="birthdate"
+          >
+        </label>
+        <label class="label" for="agreement"
+          >I accept the term of the agreement
+          <input
+            required
+            id="agreement"
+            type="checkbox"
+            name="agreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="label" for="email"
+          >E-mail:
+          <input
+            required
+            id="email"
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="label" for="password"
+          >Password:
+          <input
+            required
+            id="password"
+            type="password"
+            name="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="label inlineBlock">Do you love cats?
+          <input
+            id="cats-yes"
+            type="radio"
+            name="cats"
+          >
+          <label class="inlineBlock" for="cats-yes">Yes</label>
+          <input
+            id="cats-no"
+            type="radio"
+            name="cats"
+          >
+          <label class="inlineBlock" for="cats-no">No</label>
+        </label>
+        <label class="label" for="color"
+          >What is your favorite color?
+          <input
+            id="color"
+            type="color"
+            name="color"
+          >
+        </label>
+        <label class="label" for="bedtime"
+          >What time do you go to bed?
+          <input id="bedtime"
+            type="time"
+            step="1"
+            name="bedtime"
+          >
+        </label>
+        <label class="label" for="cars">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+            <option disabled></option>
+          </select>
+        </label>
+        <label class="label" for="rate"
+          >How do you rate our work?
+          <input
+            id="rate"
+            type="range"
+            value="0"
+            name="rate"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info</legend>
+        <label class="label" for="comments">
+          Comments:
+          <textarea
+            autocomplete="off"
+            id="comments"
+            name="comments">
```

Comment:
maintain opening and closing brackets at same indent level
```suggestion
          <textarea
            autocomplete="off"
            id="comments"
            name="comments"
          >
```